docs/application-connector/docs/08-03-register-manage-services.md

```diff
@@ -128,3 +128,46 @@ A successful response returns the ID of the registered service:
     ```
     curl https://gateway.kyma.local:{NODE_PORT}/{APP_NAME}/v1/metadata/services/{YOUR_SERVICE_ID} --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -k
     ```
+
+
+## Register API with specification URL
+
+Application Registry allows you to pass API specification in form of specification URL.
+
+To register API with specification URL replace `api.spec` with `api.specificationUrl`. Be aware that `api.spec` has higher priority than `api.specificationUrl`.
+
+```
```

Comment:
What is that? Please describe the example in one sentence, for example:
`See the example of blablabla:`

Reply:
That's the API part of the request body if the user wants to register their API with specification URL. 
```
See the example of the API part of the request body with specification URL:
...
>**NOTE** If both api.spec and api.specificationUrl are provided, api.spec will be used due to its higher priority.
```
Would that be better, @klaudiagrz ?

Reply:
That's exactly what I meant :)